window/src/os/macos/window.rs

```diff
@@ -745,6 +745,19 @@ impl WindowOps for Window {
         });
     }
 
+    #[cfg(target_os = "macos")]
+    fn set_level(&self, level: WindowLevel) {
+        Connection::with_window_inner(self.id, move |inner| {
+            inner.set_level(level as i64);
+            Ok(())
+        });
+    }
+
+    #[cfg(target_os = "macos")]
+    fn level(&self) -> Future<WindowLevel> {
+        Connection::with_window_inner(self.id, |inner| Ok(inner.level().into()))
+    }
+
```

Comment:
This file is already macos specific, so there is no need for the `cfg` attribute here

```suggestion
    fn set_window_level(&self, level: WindowLevel) {
        Connection::with_window_inner(self.id, move |inner| {
            inner.set_window_level(level);
            Ok(())
        });
    }
```